ilib-scanner.js

```diff
@@ -24,7 +24,7 @@
 
 var fs = require("fs");
 var path = require("path");
-
+var Locale = require("ilib/lib/Locale");
```

Comment:
Use ilib-locale -- you don't need all of ilib!

Reply:
There is `("ilib/lib/ilib-unpack.js");` statement in this file. https://github.com/iLib-js/ilib-scanner/blob/includeClass/ilib-scanner.js#L120
How could I get this file without npm ilib install?

Reply:
Ah yes, I forgot about that part. Never mind then!